extensions/amp-analytics/analytics-vendors-list.md

```diff
@@ -45,7 +45,11 @@ Type attribute value: `adobeanalytics`, `adobeanalytics_nativeConfig`
 
 Adds support for Adobe Analytics. More details for adding Adobe Analytics support can be found at [experienceleague.adobe.com](https://experienceleague.adobe.com/docs/analytics/implementation/other/amp.html).
 
-> ⚠️ Do not use both the `adobeanalytics` and `adobeanalytics_nativeConfig` types on the same page. If you attempt to do so, you can generate errors in the browser console and double-count visitors.
+[tip type="important"]
+
+Do not use both the `adobeanalytics` and `adobeanalytics_nativeConfig` types on the same page. If you attempt to do so, you can generate errors in the browser console and double-count visitors.
+
+[/tip]
```

Comment:
I didn't know this document lived on amp.dev, so I used syntax that would look fine on Github. It doesn't look right on amp.dev so might as well use a [tip](https://amp.dev/documentation/guides-and-tutorials/contribute/contribute-documentation/formatting/#tips):

![image](https://user-images.githubusercontent.com/254946/110367631-c6479c00-7ffc-11eb-9832-0e1df6f85be4.png)